ts/store/reducers/loading.ts

```diff
@@ -29,10 +27,10 @@ export const INITIAL_STATE: LoadingState = {
  */
 export const createLoadingSelector = (
   actions: ReadonlyArray<FetchRequestActionsType>
-): ((_: GlobalState) => boolean | null) => (state: GlobalState): boolean => {
+): ((_: GlobalState) => boolean) => (state: GlobalState): boolean => {
   // Returns true only when all actions are not loading
```

Comment:
Returns true when there's no "loading" action going on

Reply:
Sorry the description was wrong. I think a good description is `Returns true if any of the actions passed as parameter is in "loading" state`

Reply:
yes much better